tests/ui/dyn-star/llvm-old-style-ptrs.rs

```diff
@@ -0,0 +1,24 @@
+// run-pass
+// compile-flags: --target x86_64-unknown-linux-gnu -Copt-level=0 -Cllvm-args=-opaque-pointers=0
```

Comment:
You can't set `--target` here I think? The ui test suite is controlling the target.

Reply:
Technically you can set `--target`, but you need a `#![no_core]` test in that case in combination with `// needs-llvm-component: x86`.

Reply:
I'll find some other combination of flags that works then